test/utils/test-swagger-gen.ts

```diff
@@ -1,305 +1,359 @@
-        swaggerPath = `node_modules/@microsoft.azure/autorest.testserver/swagger/${swagger}`;
-      }
-
-      let autorestCommand = `autorest --add-credentials=${!!addCredentials} --typescript --output-folder=./test/integration/generated/${name} --use=. --title=${clientName} --input-file=${swaggerPath} --package-name=${packageName} --package-version=${package_version}`;
-
-      if (isDebugging) {
-        autorestCommand = `${autorestCommand} --typescript.debugger`;
-      }
-
-      const generationTask = () => {
-        return new Promise<void>((resolve, reject) => {
-          exec(autorestCommand, (error, stdout, stderr) => {
-            if (error) {
-              reject(`Failed to generate ${name} with error: \n ${error}`);
-              return;
-            }
-            console.log(`=== Start ${name} ===`);
-            console.log(stdout);
-            stderr && console.error(stderr);
-            console.log(`=== End ${name} ===`);
-            resolve();
-          });
-        });
-      };
-
-      generationTasks.push(generationTask());
-    });
-
-  try {
-    await Promise.all(generationTasks);
-  } catch (error) {
-    console.error(error);
-    throw error;
-  }
-};
-
-const buildWhitelist = () =>
-  process.argv.forEach((arg, index) => {
-    if (arg !== "--include" && arg !== "-i") {
-      return;
-    }
-
-    const includesValue = process.argv[index + 1];
-    if (!includesValue) {
-      console.warn(
-        "No valid include list provided, generating all test swaggers"
-      );
-      return;
-    }
-
-    const swaggers = includesValue.split(",");
-
-    swaggers.forEach((swagger) => {
-      const validSwaggers = Object.keys(testSwaggers);
-      if (!validSwaggers.includes(swagger)) {
-        throw new Error(
-          `Unknown swagger ${swagger}, valid swaggers: \n ${JSON.stringify(
-            validSwaggers
-          )}`
-        );
-      }
-
-      whiteList.push(swagger);
-    });
-  });
-
-const buildAutorest = () => {
-  if (!process.argv.includes("--build") && !process.argv.includes("-b")) {
-    console.warn(
-      "Not building Autorest.Typescript, use --build or -b to build"
-    );
-    return Promise.resolve();
-  }
-
-  return new Promise<void>((resolve, reject) => {
-    console.log("Building Autorest.Typescript");
-    exec("npm run build", (error, stdout, stderror) => {
-      if (error) {
-        reject(
-          `Failed to build autorest.typescript \n ${JSON.stringify(error)}`
-        );
-        return;
-      }
-
-      console.log(stdout);
-      stderror && console.error(stderror);
-      resolve();
-    });
-  });
-};
-
-const run = async () => {
-  const isDebugging = process.argv.indexOf("--debug") !== -1;
-  buildWhitelist();
-  await buildAutorest();
-  await generateSwaggers(whiteList, isDebugging);
-};
-
-run().catch((error) => {
-  process.exit(-1000);
-});
+import { exec } from "child_process";

```

Comment:
Are there any changes in this file? Or just formatting?

Reply:
Just formatting